matomo/templates/deployment.yaml

```
@@ -106,9 +106,19 @@ spec:
             {{ toYaml . | indent 12 | trim}}
         {{- end }}
           volumeMounts:
+            - name: data
+              mountPath: {{ .Values.persistence.data.mountPath }}
+              subPath: {{ .Values.persistence.data.subPath }}
```

Unnötige Einordnung unter `data`.
```suggestion
              mountPath: {{ .Values.persistence.mountPath }}
              subPath: {{ .Values.persistence.subPath }}
```